.github/workflows/npm-bundle.yml

```diff
@@ -1,7 +1,5 @@
 name: Publish Design System to npm
-on:
-  release:
-    types: [published]
+on: push
```

Comment:
Need to change this back to release

Reply:
Yes it doesn't actually work yet 😢

Reply:
Ah ok maybe I was a bit early with my review 😂